examples/v1beta1/taskruns/task-output-image.yaml

```diff
@@ -1,97 +0,0 @@
-# `PipelineResources` are deprecated, consider using `Tasks` and other replacement features instead
-# https://github.com/tektoncd/pipeline/blob/main/docs/migrating-v1alpha1-to-v1beta1.md#replacing-pipelineresources-with-tasks
-apiVersion: tekton.dev/v1alpha1
-kind: PipelineResource
-metadata:
-  name: skaffold-image-leeroy-web-output-image
-spec:
-  type: image
```

Comment:
Image pipeline resources are not really removed yet, but I guess they'll go away soon enough, so probably ok to remove this... I would have preferred a neat separation though